docs_sphinx/user/nmlimport.rst

```diff
@@ -0,0 +1,322 @@
+NeuroML importer (nmlimport library)
+================
+
+This is a short overview of the `~brian2tools.nmlimport` package, providing
+functionality to import Brian's morphology from a .nml file.
+
+NeuroML is a XML-based description that provides a common data format
+for defining and exchanging descriptions of neuronal cell and network models
+(`NML project website <https://neuroml.org/>`_).
+
+.. contents::
+    Overview
+    :local:
+
+Working example
+---------------
+
+As a demonstration, we will use nmlimport library to generate morphology and
+extract other related information from the ``pyramidal cell's`` nml file. You
+can download it from `here <https://github
+.com/OpenSourceBrain/ACnet2/blob/master/neuroConstruct/generatedNeuroML2/pyr_4_sym.cell.nml>`_.
+
+
+.. code:: python
+
+    from brian2tools.nmlimport.nml import NmlMorphology
+    nml_object = NmlMorphology('pyramidal.cell.nml',name_heuristic=True)
+
+This call provides us the ``nml_object`` that contains all the information
+extracted from `.nml` file. When ``name_heuristic`` param is set to True
+morphology sections will be determined based on the segment names and section
+name will be created by combining names of the inner segments of the section
+else if set to False, all linearly connected segments combines and form a
+section and naming convention `sec{unique integer}` is followed.
+
+|
+- To obtain morphology object.
+
+.. code:: python
+
+    morphology=nml_object.morphology_obj
+|
+
+- To obtain topology information.
```

Comment:
Maybe this section could be shortened a bit. How about saying: "With this `Morphology` object, you can use all of Brian's functions to get information about the cell:" and then having a long code block (with these interactive `>>>` prompts). I don't think you need to say that `morphology.area` gives you the area, etc.